test/EFCore.PG.Tests/Storage/NpgsqlNetTopologySuiteTypeMappingSourcePluginTest.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using GeoAPI.Geometries;
+using Microsoft.EntityFrameworkCore.Storage;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Internal;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal;
+using Xunit;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage
+{
+    public class NpgsqlNetTopologySuiteTypeMappingSourcePluginTest
+    {
+        [Theory]
+        [InlineData(typeof(IPoint), "GEOMETRY (POINT)")]
+        [InlineData(typeof(IGeometry), "GEOMETRY")]
+        [InlineData(typeof(string), null)]
+        public void Geom_by_ClrType(Type clrType, string storeType)
+            => Assert.Equal(storeType, _geomPlugin.FindMapping(new RelationalTypeMappingInfo(clrType))?.StoreType);
+
+        [Theory]
+        [InlineData(typeof(IPoint), "GEOGRAPHY (POINT)")]
+        [InlineData(typeof(IGeometry), "GEOGRAPHY")]
+        [InlineData(typeof(string), null)]
+        public void Geog_by_ClrType(Type clrType, string storeType)
+            => Assert.Equal(storeType, _geogPlugin.FindMapping(new RelationalTypeMappingInfo(clrType))?.StoreType);
+
+        [Theory]
+        [InlineData("GEOMETRY (POINT)", typeof(IPoint))]
+        [InlineData("GEOGRAPHY (POINT)", typeof(IPoint))]
+        [InlineData("GEOMETRY", typeof(IGeometry))]
+        [InlineData("text", null)]
+        public void By_StoreType(string storeType, Type clrType)
```

Comment:
nit: can you reorder the types/strings above like the other tests?

Reply:
I'm not sure I understand - are you referring to the method parameters? If so the idea was for the 1st param to be the input, and for the 2nd to be the expected output... Do you see an issue, or prefer it some other way?